Mathlib/Control/Functor/Multivariate.lean

```diff
@@ -121,7 +121,7 @@ section LiftP'
 variable (F)
 
 theorem exists_iff_exists_of_mono {P : F α → Prop} {q : F β → Prop}
-                                  (f : α ⟹ β) (g : β ⟹ α)
+    (f : α ⟹ β) (g : β ⟹ α)
```

Comment:
Could you either undo this or fix the rest of the statement?

Reply:
done